node/src/scss/_dataviz.scss

```diff
@@ -0,0 +1,32 @@
+.bar {
```

Comment:
I noticed this is included with the npm package, and `node/build/vendor.js` can handle the bundling of it with other vendor styles. Include the path on line 18.